app/src/main/java/com/spendster/presentation/mainScreen/ActivityFragment.java

```diff
@@ -0,0 +1,65 @@
+package com.spendster.presentation.mainScreen;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.support.v4.view.ViewPager;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.spendster.R;
+import com.spendster.presentation.welcomeScreen.SlideFragment;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ActivityFragment extends Fragment {
```

Comment:
@SimonMaha bad name, it's not clear as we already have system name "Activity" in android